api/tests/identity-access-management/acceptance/application/token.route.test.js

```diff
@@ -143,6 +143,42 @@ describe('Acceptance | Identity Access Management | Route | Token', function ()
         expect(result.user_id).to.equal(userId);
         expect(result.refresh_token).to.exist;
       });
+
+      context('when the scope is different from the refresh token one', function () {
+        it('returns a 401 (unauthorized)', async function () {
+          // given
+          const { result: accessTokenResult } = await server.inject({
+            method: 'POST',
+            url: '/api/token',
+            headers: {
+              'content-type': 'application/x-www-form-urlencoded',
+            },
+            payload: querystring.stringify({
+              grant_type: 'password',
+              username: userEmailAddress,
+              password: userPassword,
+              scope: 'pix-orga',
+            }),
+          });
+
+          // when
+          const response = await server.inject({
+            method: 'POST',
+            url: '/api/token',
+            headers: {
+              'content-type': 'application/x-www-form-urlencoded',
+            },
+            payload: querystring.stringify({
+              grant_type: 'refresh_token',
+              refresh_token: accessTokenResult.refresh_token,
+              scope: 'pix-admin',
```

Comment:
Pour rendre le cas plus évident et explicite : 

```suggestion
              scope: 'pix-app',
            }),
          });

          // when
          const differentScope = 'pix-admin'
          const response = await server.inject({
            method: 'POST',
            url: '/api/token',
            headers: {
              'content-type': 'application/x-www-form-urlencoded',
            },
            payload: querystring.stringify({
              grant_type: 'refresh_token',
              refresh_token: accessTokenResult.refresh_token,
              scope: differentScope,
```